src/utils/bridge/sub/classes/adapter.ts

```diff
@@ -134,45 +161,14 @@ class RococoAdapter extends SubAdapter {
     );
   }
 
+  /* Throws error until Substrate 5 migration */
```

Comment:
But it doesn't
Please update this comment. You can use JS Doc

Reply:
It throws error, but i hardcoded result in `catch `clause